test/browser/step_definitions/F2FBackendValidation.step.js

```diff
@@ -0,0 +1,69 @@
+
+const { Given, When, Then } = require("@cucumber/cucumber");
+
+const { expect } = require("chai");
+
+// const { axios } = require("axios");
+
+
+const TestHarness = require("../support/TestHarness");
+Given(/^I have retrieved the sessionTable data for my F2F session$/, { timeout: 2 * 50000 }, async function () {
+  await new Promise(r => setTimeout(r, 10000));
+
+  const testHarness = new TestHarness();
+  const authCodeDetails = await testHarness.getSessionByAuthCode(this.authCode);
+  expect(authCodeDetails.authorizationCode).to.equal(this.authCode);
+  this.sessionId = authCodeDetails.sessionId
+  const session = await testHarness.getSession(this.sessionId);
+  this.authSessionState = session.authSessionState;
+
+});
+
+
+Then(/^session details are correctly stored in DB$/, { timeout: 2 * 50000 }, async function () {
+  expect(this.sessionId).to.not.be.null;
+  expect(this.authSessionState).to.equal("F2F_AUTH_CODE_ISSUED");
+});
+
+When(/^I sent the request to the callback endpoint$/, { timeout: 2 * 50000 }, async function () {
+  const testHarness = new TestHarness();
+  const f2fSession = await testHarness.getSession(this.sessionId);
+  this.yotiSessionId = f2fSession.yotiSessionId;
+  this.subject = f2fSession.subject;
+  console.log(this.yotiSessionId);
+  const axios = require("axios");
+  const postRequest = await axios.post("https://api-f2f-cri-api.review-o.dev.account.gov.uk/callback", {
```

Comment:
Is there any benefit to using an env var here so that they tests can easily be pointed at another environment if needed?